src/braket/circuits/angled_gate.py

```diff
@@ -97,8 +99,15 @@ def adjoint(self) -> List[Gate]:
         Returns:
             List[Gate]: A list containing the gate with negated angle.
         """
+        gate_ascii_name_index = self.ascii_symbols[0].find("(")
+        gate_ascii_name = self.ascii_symbols[0][:gate_ascii_name_index]
```

Comment:
Looks a bit hacky. Is it better to create a brand new gate instance rather than replacing properties of a copied instance?

Reply:
It's a little tricky since you're creating a subclass instance from a superclass instance so you don't have any guarantees about the constructor arguments etc. This approach is not introduced with this PR, just updated to update one property that was missed